Unity-Technologies/ScriptableRenderPipeline
# Optimized build time by decreasing shader compilation/variants.

### Purpose of this PR
 - Optimized shader build time by spliting shader variants between vertex and fragment. This provides a big improvement in terms of build times.
 - Added `local` shader compilation decoration to some variants. 

### Testing status

**Manual Tests**: What did you do?
- [x ] Built a player
- [x ] C# and shader warnings (supress shader cache to see them)

**Automated Tests**: Not required

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline

---
### Comments to reviewers
 Currently there's a known issue with multi_compile_local and OnPreProcessShaders. Therefore I only added _local to multi_compile in post processing.
